### PR TITLE
Create onboarding

### PR DESCRIPTION
**Pull request by kakashii7**
create onboarding page and image for onboarding page and css file

### CODE REVIEW

**Review by ahmedirheem — DISMISSED** [review]
Good job

File: assets/css/style.css [hunk 1]
```diff
@@ -0,0 +1,103 @@
+        body {
+            display: flex;
+            flex-direction: column;
+            background-color: #94cfb2
+        }
```
Comment (sara219) [inline assets/css/style.css]: there's a file called style.css in the root, why did you create a new one

File: index.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="assets/css/style.css">
+
+    <body>
+        <img id="unboardingimg" src=" ./assets/image/unboarding.svg " alt=" ">
+        <div class="container">
+            <p id="managetext">Manage Your Task</p>
+
+            <input id="name" onselect="fun(this)" type="text" name=" " placeholder="Enter your name ">
```
Comment (sara219) [inline index.html:16]: I think it needs to be more organized, the title, input and button to be in the same level and direction


Screen Shot 2021-11-11 at 12 23 32 PM

**Review by sara219 — CHANGES_REQUESTED** [review]
the assets folder provide a way to add arbitrary files like text, fonts, music, and video in the application, we don't add style and js files into it 
remove the CSS and JS folders from it and put them in the root

**Review by ahmedirheem — DISMISSED** [review]

**Review by farahalzard — APPROVED** [review]

**Review by ahmedirheem — APPROVED** [review]

**Review by sara219 — APPROVED** [review]

**Pull request merged**